### PR TITLE
Update move-users-different-subscription.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to Move users to a different sub as options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4371

File: microsoft-365/commerce/subscriptions/move-users-different-subscription.md [hunk 1]
```diff
@@ -44,11 +44,9 @@ You must be a Global, License, or User admin to assign licenses. For more inform
 
 1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.
 2. Select the circles next to the names of the users that you want to replace existing licenses for.
-3. At the top, select **More options (...)**, then select **Manage product licenses**.
-4. In the **Manage product licenses** pane, select **Replace existing product license assignments** \> **Next**.
-5. Switch the toggle to the **On** position for the licenses that you want to assign to these users.\
-    You can limit which services are available to the users. Switch the toggles to the **Off** position for the services that you don't want those users to have. Any previous license assignments for the selected users are removed.
-6. At the bottom of the **Replace existing products** pane, select **Replace** \> **Close**.
+3. At the top, select **Manage product licenses**.
+4. In the **Manage product licenses** pane, Select the licenses that you would like to assign to the user and Unselect the current sibscription.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/move-users-different-subscription.md]:
```suggestion
4. In the **Manage product licenses** pane, select the licenses that you would like to assign to the user and unselect the current subscription.
```

Comment (JohanFreelancer9) [reply microsoft-365/commerce/subscriptions/move-users-different-subscription.md]: Shouldn't the current subscription be unselected before the new licenses are assigned?

File: microsoft-365/commerce/subscriptions/move-users-different-subscription.md [hunk 2]
```diff
@@ -44,11 +44,9 @@ You must be a Global, License, or User admin to assign licenses. For more inform
...
+5. At the bottom , select **Save Changes** \> **Close**.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/move-users-different-subscription.md]:
```suggestion
5. At the bottom, select **Save Changes** \> **Close**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks.

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
The steps that have been added are for 1 user only. Please rewrite so that it covers multiple users (the UI is slightly different in that case).
Also, it's "deselect" not "unselect". Please fix.

**Comment by yogkumgit** [comment]
@msbemba Please implement the feedback. Thanks

**Comment by msbemba** [comment]
@cmcatee-MSFT Thank you for the input. I have made changes to the PR to reflect the same.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**